src/tiles/grid/item-tile.ts

```diff
@@ -15,9 +17,11 @@ export class ItemTile extends LitElement {
 
   @property({ type: String }) baseImageUrl?: string;
 
+  @property({ type: Object }) sortParam: SortParam | null = null;
```

Comment:
```suggestion
  @property({ type: Object }) sortParam?: SortParam;
```
denotes that it is optional and can default to undefined